sycl/include/sycl/ext/intel/esimd/detail/memory_intrin.hpp

```diff
@@ -348,6 +348,36 @@ __esimd_svm_atomic2(__ESIMD_DNS::vector_type_t<uint64_t, N> addrs,
 }
 #endif // __SYCL_DEVICE_ONLY__
 
+/// USM pointer atomic.
+/// Supported platforms: DG2, PVC
+///
+/// @tparam Ty is element type.
+/// @tparam InternalOp is operation type.
+/// @tparam L1H is L1 cache hint.
+/// @tparam L3H is L3 cache hint.
```

Comment:
I think we've been renaming this to L2H when moving to supported

Reply:
Done in https://github.com/intel/llvm/pull/11833/commits/ffe19702c45355afd8b7ab17d2170cd783e9f670